app/client/components/callCentreNumbers.tsx

```diff
@@ -1,11 +1,11 @@
+import { css } from "@emotion/core";
 import React from "react";
```

Comment:
I notice that you're still importing `React` into `*.tsx` files. Is this still necessary? I believe JSX should [transpile down to calls to `@emotion/core`'s `jsx`](https://emotion.sh/docs/@emotion/babel-preset-css-prop#features), which  is auto-imported by `@emotion/babel-preset-css-prop`. Is this correct?

Reply:
Interesting, if I remove that though I get some compilation issues, I wonder also whether it's worth leaving in for expressiveness to make clear that React is being used.

Reply:
Thanks for confirming that it isn't just me receiving these compilation issues. I think it's due to a bug in `@babel/plugin-transform-react-jsx` (fixed in babel/babel#9095), although this fix has yet to roll out to `@babel/preset-typescript` (proposed fix in babel/babel#9497).

My understanding is `@emotion/babel-preset-css-prop` implicitly adds the [`jsx`](https://emotion.sh/docs/@emotion/babel-preset-css-prop#features) function and `/** @jsx jsx */` pragma to each module, so `React` isn't used directly in the module. I think explicitly importing React is misleading, since it's not actually being used. However, we still need to import it to keep TypeScript happy, which feels a bit wonky.

In dotcom-rendering, I think we're going to keep importing React until the above issue in the Babel TypeScript preset is fixed, then hopefully we can remove it.

Reply:
thanks for the extra explanation.

In particular...
> In dotcom-rendering, I think we're going to keep importing React until the above issue in the Babel TypeScript preset is fixed, then hopefully we can remove it.

...I think we'll do the same with `manage-frontend`

Reply:
Hi @twrichards 

babel/babel#9497 has been merged, so you should be able to upgrade `@babel/preset-typescript` and remove the React imports, if you want to.

After some careful consideration, we've decided not to migrate to using the CSS prop, so we're sticking with the vanilla `emotion` API (`css` to generate classnames, `cx` for composition). I'll write a more detailed explanation elsewhere, but the short version is that the new API is optimised for universal React or running on the client, which we is an approach we are no longer taking. We have already encountered friction styling AMP elements, which uses a combination of custom SSR components and non-React client side presentational logic.